mmv1/third_party/terraform/tests/resource_compute_instance_group_manager_test.go.erb

```diff
@@ -1390,7 +1390,7 @@ resource "google_compute_instance_group_manager" "igm-basic" {
   target_size        = 2
   stateful_disk {
     device_name = "my-stateful-disk"
-    delete_rule = "NEVER"
+    delete_rule = "ON_PERMANENT_INSTANCE_DELETION"
```

Comment:
Can we also prefix the `base_instance_name` with `tf-test-`?

Reply:
re added delete rule and changed prefix... I think it makes sense to still check at least once that this field is not broken